README.md

```diff
@@ -1,229 +1,67 @@
-```
-./azcopy cp "https://myaccount.blob.core.windows.net/mycontainer/*?sastokenhere" /data/ --recursive=true
-```
+:white_check_mark: Recover from failures by restarting previous jobs.
 
-The following command will download all txt files in the source to the `directory1` path. Note that AzCopy will scan the entire source and filter for `.txt` files. This may take a while when you have thousands/millions of files in the source.
-```
-./azcopy cp "https://myaccount.blob.core.windows.net/mycontainer/directory1/*.txt?sastokenhere" /data/directory1
-```
+## Find help
 
-### Copy data between Azure Storage accounts (currently supports Block Blobs only)
+For complete guidance, visit any of these articles on the docs.microsoft.com website.
 
-Copying data between two Azure Storage accounts make use of the PutBlockFromURL API, and does not use the client machine's network bandwidth. Data is copied between two Azure Storage servers. AzCopy simply orchestrates the copy operation.
-```
-./azcopy cp "https://myaccount.blob.core.windows.net/?sastokenhere" "https://myotheraccount.blob.core.windows.net/?sastokenhere" --recursive=true
-```
+:eight_spoked_asterisk: [Get started with AzCopy](https://docs.microsoft.com/azure/storage/common/storage-use-azcopy-v10)
 
-### Advanced Use Cases
+:eight_spoked_asterisk: [Transfer data with AzCopy and blob storage](storage-use-azcopy-blobs.md)
 
-#### Configure Concurrency
+:eight_spoked_asterisk: [Transfer data with AzCopy and file storage](storage-use-azcopy-files.md)
 
-Set the environment variable `AZCOPY_CONCURRENCY_VALUE` to configure the number of concurrent requests. This is set to 300 by default. Note that this does not equal to 300 parallel connections. Reducing this will limit the bandwidth, and CPU used by AzCopy.
+:eight_spoked_asterisk: [Transfer data with AzCopy and Amazon S3 buckets](storage-use-azcopy-s3.md)
 
-#### Configure proxy settings
-To configure the proxy settings for AzCopy v10, set the environment variable https_proxy using the following command:
+:eight_spoked_asterisk: [Configure, optimize, and troubleshoot AzCopy](storage-use-azcopy-configure.md)
 
-```
-# For Windows:
-set https_proxy=<proxy IP>:<proxy port>
-# For Linux:
-export https_proxy=<proxy IP>:<proxy port>
-# For MacOS
-export https_proxy=<proxy IP>:<proxy port>
-```
+### Find help from your command prompt
 
-For proxy authentication, use the following format:
+Be sure to add AzCopy to your system path. Then, open a command prompt window.
 
-```
-export https_proxy=<user>:<pass>@<proxy IP>:<proxy port>
-# or with a domain:
-export https_proxy=<domain>%5C<user>:<pass>@<proxy IP>:<proxy port>
-```
+To see a list of commands, type `azcopy` and then press the ENTER key.
 
-### Configure log location
+To learn about a specific command, type `azCopy` followed by the name of the command.
```

Comment:
should be `azcopy cp --help` or `azcopy cp -h` instead. If `azcopy command` is called without any args, an error message is printed.